CMakeLists.txt

```diff
@@ -85,6 +85,90 @@ if(OGS_BUILD_GUI)
 endif()
 option(OGS_BUILD_UTILS "Should the utilities programms be built?" OFF)
 
+
+# A list of processes to be build.
+option(OGS_BUILD_PROCESS_ComponentTransport "Build the component transport process." ON)
+if(OGS_BUILD_PROCESS_ComponentTransport)
+    add_definitions(-DOGS_BUILD_PROCESS_COMPONENTTRANSPORT)
```

Comment:
@bilke Shall I make the cmake option capital case too, same as the c-definitions? All capital case are harder to read, but would be consistent with the c-definitions.

Reply:
I do not know whats best but for me the mixed case you did is better to read indead..

Reply:
OK. Keeping mixed case.